apps/masterbots.ai/app/actions/ai-executers.ts

```
@@ -0,0 +1,275 @@
+    inputs = data.inputs.reduce(
+      (acc: any, input: { label: any }) => ({
+        ...acc,
+        [input.label]: ''
+      }),
+      {}
+    )
+  } catch (error) {
+    console.error('Error fetching prompt details:', error)
+    error = (error as Error).message
+  } finally {
+    return { data, error, inputs }
```

Biome

[error] Unsafe usage of 'return'.

'return' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)